private/model/api/eventstream.go

```diff
@@ -276,7 +279,10 @@ func setupEventStream(s *Shape) *EventStream {
 
 		updateEventPayloadRef(eventRef.Shape)
 
-		eventRef.Shape.EventFor = append(eventRef.Shape.EventFor, eventStream)
+		if eventRef.Shape.EventFor == nil {
+			eventRef.Shape.EventFor = map[string]*EventStream{}
+		}
+		eventRef.Shape.EventFor[eventStream.Name] = eventStream
```

Comment:
Just to double check, what is sorting the event names?

Reply:
Relying on behavior of template's range helper